### ISSUE TITLE
Why does this repo use "Apache License 2.0" instead of  "Creative Commons CC0 1.0 Universal (CC-0)" or "BSD 0-Clause License (0BSD)"?

### ISSUE THREAD

**Issue by hab25** [GoogleCloudPlatform/nodejs-docs-samples#2487]
The reason I believe one of the two alternatives would be superior is that they don't require including licenses or copyright notices.

The GCP documentation has many examples where it instructs readers to copy/paste code from sourced from this repo, e.g.
https://cloud.google.com/billing/docs/how-to/notify#write-a-cloud-function-2
Doesn't this lead readers to inadvertently break the Apache License 2.0 by doing so without including the license, copyright notices or stating changes in their code?

TL;DR of the three mentioned licenses:
https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)
https://tldrlegal.com/license/creative-commons-cc0-1.0-universal
https://tldrlegal.com/license/bsd-0-clause-license

**Comment by SurferJeffAtGoogle** [comment]
Google's standard license for code and code-related documentation is Apache 2.0.  We believe it best protects the rights of both the authors and users. Googlers aren't permitted to share code under terms that would give up copyright ownership.
